python/res/job_queue/job_queue_manager.py

```diff
@@ -58,80 +55,62 @@ def __init__(self, queue):
         self.queue = queue
         super(JobQueueManager, self).__init__(c_ptr)
 
-    def status_timestamp(self):
-        """
-        Will return a datetime object corresponding to the last status change
-        """
-        ts = self._status_timestamp()
-        return ts.datetime()
-
-    def progress_timestamp(self, job_index = None):
-        """Will return the timestamp of last progress update.
-
-        By default the timestamp will be global, i.e. for all jobs, but if you
-        pass a job_index as optional argument the timestamp will correspond to
-        that job.
-        """
-        if job_index is None:
-            ts = self._global_progress_timestamp()
-        else:
-            ts = self._iget_progress_timestamp(job_index)
-
-        return ts.datetime()
-
-
     def get_job_queue(self):
         return self.queue
 
     def stop_queue(self):
-        self._stop_queue( )
+        self.get_job_queue().kill_all_jobs()
```

Comment:
If you have managed to remove all the calls to the C methods, please remove also the corresponding cwrap declarations at the beginning of the file

Reply:
There is a new issue for it: #728